src/probabilities.jl

```diff
@@ -178,11 +181,23 @@ Works for estimators that implement [`total_outcomes`](@ref).
 
 See also: [`MissingDispersionPatterns`](@ref).
 """
-function missing_outcomes end
-
 function missing_outcomes(x::Array_or_Dataset, est::ProbabilitiesEstimator)
     probs = probabilities(x, est)
     L = total_outcomes(x, est)
     O = count(!iszero, probs)
     return L - O
 end
+
+"""
+    all_possible_outcomes([x,] est::ProbabilitiesEstimator) → o
+
+Return a vector containing all possible outcomes of `est`.
```

Comment:
I'd say *"Return a vector containing the set of all possible outcomes, i.e. the outcome space, for `est`."*

Reply:
> I am thinking to not restrict outcome_space to return Vector no matter what. For example, for the binning here the most efficient return is simply the cartesian indices in array form. Right? For most estimators the return type is indeed a vector.

We should return whatever is most efficient for a particular estimator. So yes, this can be reworded to not be strict about a vector being returned.